docs/src/ci.md

```diff
@@ -118,7 +118,7 @@ jobs:
         PLAYWRIGHT_TEST_BASE_URL: ${{ github.event.deployment_status.target_url }}
 ```
 
-### Docker
+## Docker
```

Comment:
Yeah this was a little confusing to me since the two GHA examples where non-docker then after the Docker section (at the same heading level) it goes into circle, jenkins and the rest all using docker. Maybe this can but separated into logical sections.